assets/css/fontawesome.scss

```diff
@@ -2,12 +2,7 @@
 /* Font Awesome Icons */
 @font-face {
 	font-family: 'FontAwesomeSensei';
-	src: url('./fonts/fontawesome-webfont.eot');
-	src: url('./fonts/fontawesome-webfont.eot?#iefix') format('embedded-opentype'),
-	url('./fonts/fontawesome-webfont.woff2') format('woff2'),
-	url('./fonts/fontawesome-webfont.woff') format('woff'),
-	url('./fonts/fontawesome-webfont.ttf')  format('truetype'),
-	url('./fonts/fontawesome-webfont.svg#svgFontName') format('svg');
+	src: url('./fonts/fontawesome-webfont.woff') format('woff');
```

Comment:
Would be better to keep the woff2 too since it's more modern than the woff? At some point, I think we could remove the woff that would be there only for IE support for now.

Reply:
Yeah, looks like they are also a bit smaller. Adding them back in c702485c.